src/proxyvisor.coffee

```diff
@@ -146,7 +146,7 @@ class ProxyvisorRouter
 			.then ([ device ]) =>
 				return res.status(404).send('Device not found') if !device?
 				return res.status(410).send('Device deleted') if device.markedForDeletion
-				@logger.log(m, { channel: "device-#{device.logs_channel}-logs" })
+				@logger.logDependent(m, { uuid, channel: "device-#{device.logs_channel}-logs" })
```

Comment:
The `channel` key is unused in `logDependent`, so this can just be `@logger.logDependent(m, { uuid })`

Reply:
Not really @LucianBuzzo - it's unused in the native logger's `logDependent` but used by the pubnub logger's `logDependent`, so we still need it.

Reply:
okay cool